webapp/channels/src/components/admin_console/manage_roles_modal/manage_roles_modal.tsx

```diff
@@ -46,6 +49,7 @@ function getStateFromProps(props: Props): State {
     const roles = props.user && props.user.roles ? props.user.roles : '';
 
     return {
+        show: true,
```

Comment:
cc @devinbinnie

Reply:
Yes we do because we will be opening the modal for a new user each time, even if it's the same user twice we always want `show` to be `true` unless we're closing the modal